src/DssSpell.sol

```diff
@@ -52,224 +72,204 @@ contract DssSpellAction is DssAction {
+    address internal constant STUSDS_RATE_SETTER_BUD = 0xBB865F94B8A92E57f79fCc89Dfd4dcf0D3fDEA16;
+
+    address internal immutable ILK_REGISTRY = DssExecLib.reg();
+    address internal immutable MCD_VAT = DssExecLib.vat();
+    address internal immutable DAI = DssExecLib.dai();
+    address internal immutable REWARDS_DIST_USDS_SKY = DssExecLib.getChangelogAddress("REWARDS_DIST_USDS_SKY");
+    address internal immutable MCD_VEST_SKY_TREASURY = DssExecLib.getChangelogAddress("MCD_VEST_SKY_TREASURY");
+    address internal immutable SKY = DssExecLib.getChangelogAddress("SKY");
+    address internal immutable DAI_USDS = DssExecLib.getChangelogAddress("DAI_USDS");
 
     // ---------- Wallets ----------
-    address internal constant AAVE_V3_TREASURY         = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c;
-    address internal constant BLUE                     = 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf;
-    address internal constant BONAPUBLICA              = 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3;
-    address internal constant CLOAKY_2                 = 0x9244F47D70587Fa2329B89B6f503022b63Ad54A5;
-    address internal constant CLOAKY_KOHLA_2           = 0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a;
-    address internal constant EXCEL                    = 0x0F04a22B62A26e25A29Cba5a595623038ef7AcE7;
-    address internal constant FORTIFICATION_FOUNDATION = 0x483413ccCD796Deddee88E4d3e202425d5E891C6;
-    address internal constant SKY_FRONTIER_FOUNDATION  = 0xca5183FB9997046fbd9bA8113139bf5a5Af122A0;
-    address internal constant LIQUIDITY_BOOTSTRAPPING  = 0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD;
-    address internal constant PBG                      = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
-    address internal constant TANGO                    = 0xB2B86A130B1EC101e4Aed9a88502E08995760307;
-    address internal constant WBC                      = 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47;
-
-    // ---------- Grove Proxy Spell ----------
-    // Note: The deployment address for the Grove Proxy can be found at https://forum.sky.money/t/technical-scope-of-the-star-2-allocator-launch/26190
-    address internal constant GROVE_PROXY = 0x1369f7b2b38c76B6478c0f0E66D94923421891Ba;
-    address internal constant GROVE_SPELL = 0xFa533FEd0F065dEf8dcFA6699Aa3d73337302BED;
+    address internal constant LIQUIDITY_BOOTSTRAPPING = 0xD8507ef0A59f37d15B5D7b630FA6EEa40CE4AFdD;
+    address internal constant ECOSYSTEM_TEAM = 0x05F471262d15EECA4059DadE070e5BEd509a4e73;
 
     // ---------- Spark Proxy Spell ----------
     // Note: Spark Proxy: https://github.com/sparkdotfi/sparklend-deployments/blob/bba4c57d54deb6a14490b897c12a949aa035a99b/script/output/1/primary-sce-latest.json#L2
     address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0xa57d3ea3aBAbD57Ed1a1d91CD998a68FB490B95E;
+    address internal constant SPARK_SPELL = 0xe7782847eF825FF37662Ef2F426f2D8c5D904121;
```

Comment:
Confirm this matches the payload reviewed by the Spark team: https://github.com/sparkdotfi/spark-spells/blob/4904b8d00e7c03e9e91bc0ed9e0c5aa8e7a2746f/src/proposals/20250904/SparkEthereum_20250904.t.sol#L44